### PR TITLE
Open feign implementation

### PR DESCRIPTION
**Pull request by kleyton-ohare**
This is to contemplate the Unosquare Action Item to add a Feign Client functionality.

There is no need to spin up docker containers to test this functionality. Once the program is running, make a GET request to:
http://localhost:8086/v1/journeys/calculateDistance?locationFrom=BT62-3JR&countryFrom=GBR&locationTo=BT71-6SS&countryTo=GBR

The JSON response body should match https://www.distance.to/BT62-3JR,GBR/BT71-6SS,GBR

### CODE REVIEW

File: server/src/main/java/com/unosquare/carmigo/service/JourneyService.java [hunk 1]
```diff
@@ -97,4 +110,49 @@ private Journey findJourneyById(final int id) {
...
+  private String prepareRequestToDistanceApi(final CreateCalculateDistanceCriteria criteria) {
+    final StringBuilder request = new StringBuilder("[{\"t\":");
+    request.append("\"").append(criteria.getLocationFrom()).append(",").append(criteria.getCountryFrom()).append("\"");
+    request.append("},{\"t\":");
+    request.append("\"").append(criteria.getLocationTo()).append(",").append(criteria.getCountryTo()).append("\"");
+    request.append("}]");
+    return request.toString();
+  }
+
+  private GrabDistanceDTO convertDistanceHolderToDistanceViewModel(final DistanceHolder distanceHolder) {
```
Comment (chrissimpson-unosquare) [inline server/src/main/java/com/unosquare/carmigo/service/JourneyService.java:123]: Hi Kleyton!

Do you think you could possibly reduce the size of this method?

I always try and follow the rule of 7.  7 lines per method, 7 methods per class, 7 classes per package.  But that's just me :)

**Review by chrissimpson-unosquare — COMMENTED** [review]
All looks good Kleyton!

The only comment I had was on the big method that you could consider breaking down.

**Review by linda-mccall — CHANGES_REQUESTED** [review]
left a small suggestion, let me know if you have any questions

**Review by linda-mccall — APPROVED** [review]
looks good 👍

**Pull request merged**